variables.tf

```
@@ -18,11 +18,12 @@ variable "include_tags" {
 
 variable "monitors" {
   type = map(object({
-    name       = string
-    message    = string
-    query      = string
-    thresholds = map(string)
-    type       = string
+    name                      = string
+    message                   = string
+    monitor_threshold_windows = map(string)
```

I would suggest to make an object out of this:

```suggestion
    monitor_threshold_windows = object({
    trigger_window = string
    recovery_window    = string
  })
```